### PR TITLE
Create deg2rad.md

### LINKED ISSUE #5246: [Term Entry] Python NumPy - Math functions .deg2rad()
### Reason/inspiration (optional)

We would like a new term entry on the `.deg2rad()` function of the [`math-methods`](https://www.codecademy.com/resources/docs/numpy/math-methods) concept in NumPy, Python. The entry should go in a new file under `docs/content/numpy/concepts/math-methods/terms/deg2rad/deg2rad.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Good-Soma**
### Issue Solved
Closes #5246 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Good-Soma thank you for contributing to Codecademy Docs 😄 

Could you please add a ##Codebyte Example block at the end of this entry that showcases the term in the entry. Thank you!

**Comment by mamtawardhani** [comment]
Hey @Good-Soma could you please remove the mixedlm entry from this branch? This is supposed to be only for the deg2rad.md file

**Comment by Good-Soma** [comment]
> Hey @Good-Soma thank you for contributing to Codecademy Docs 😄
> 
> Could you please add a ##Codebyte Example block at the end of this entry that showcases the term in the entry. Thank you!

Okay, that means I will have to create a separate branch for the mixedlm content?

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @Good-Soma  😄 

The entry looks good for a second round of review! 🚀

File: content/numpy/concepts/math-methods/terms/deg2rad/deg2rad.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  - 'Web Development'
+Tags:
+  - 'Math'
+  - 'NumPy'
+CatalogContent:
+- 'learn-python-3'
+- 'paths/computer-science'
+---
+
+In NumPy, the **`.deg2rad()`** function converts angles from degrees to radians.
```
Comment (PragatiVerma18) [inline content/numpy/concepts/math-methods/terms/deg2rad/deg2rad.md:16]: To make it clearer, we should rephrase the first sentence to:

```suggestion
In NumPy, the **`.deg2rad()`** function converts an angle from degrees to radians.
```

File: content/numpy/concepts/math-methods/terms/deg2rad/deg2rad.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+> **Note:** In NumPy, the default unit for angles is radians. Therefore, the `.deg2rad()` function converts degree inputs to radians when applied.
```
Comment (PragatiVerma18) [inline content/numpy/concepts/math-methods/terms/deg2rad/deg2rad.md:18]: Second sentence can be simplified as follows:

```suggestion
> **Note:** In NumPy, the default unit for angles is radians. Therefore, the `.deg2rad()` function is used to convert angle values from degrees to radians.
```

**Comment by Good-Soma** [comment]
Okay, thanks @mamtawardhani and @PragatiVerma18 for reviewing, editing the document for clarity and finally merging it. It is really blissful to work with you!

I have completed the survey and filled out all the necessary details in it as you requested.

**Pull request merged**